site/gdocs/components/People.tsx

```diff
@@ -3,10 +3,24 @@ import * as React from "react"
 
 export default function People({
     className,
+    columns,
     children,
 }: {
     className?: string
+    columns?: "2"
     children: React.ReactNode
 }) {
-    return <div className={cx("people", className)}>{children}</div>
+    return (
+        <div
+            className={cx(
+                "people",
+                {
+                    "people-cols-2": columns === "2",
```

Comment:
If I'm understanding the requirements correctly based on what it's currently doing when i test the article on staging, when `columns: 2` is set, we want:

lg - 1 column
md - 2 coumns
sm - 1 column

if so, could you do something like:

```
const columnClassNames = {
1: "grid-cols-1",
2: "grid-sm-cols-1 grid-md-cols-2 grid-lg-cols-1"
}
<div className={cx("people grid", columnClassNames[columns])}>
  ...children
</div>
```

And then you don't have to worry about flex or that calc statement or anything. 🙂

Reply:
Yep, you are totally right. I somehow got the wrong impression about how ordering of elements changes in a grid when you change from two columns to a single column. Also, it should be just `lg-up` instead of `lg-only`. Thanks for being persistent. 🙂 Fortunately it's a small change and didn't cost us too much time.